client/src/components/Auth/Login/Login.jsx

```diff
@@ -11,19 +11,16 @@ export class Login extends React.Component {
         <p>
           Don&apos;t have an account? <Link to="/register">Register</Link>
         </p>
-        {!this.props.errors.incomplete && (
-          <div
-            className="notification is-danger"
-            data-testid="incompletenotice"
-          >
+        {!this.props.inputValidation.complete && (
```

Comment:
I don't think you should store a separate flag for whether the rest of the fields are valid or not. This is data duplication and is at risk of becoming out of sync. I would call a function that returns a boolean based on the states of all other fields in the form instead.